client/my-sites/patterns/components/pattern-gallery/client.tsx

```diff
@@ -49,6 +50,8 @@ export const PatternGalleryClient: PatternGalleryFC = ( props ) => {
 							getPatternPermalink={ getPatternPermalink }
 							key={ pattern.ID }
 							pattern={ pattern }
+							patternTypeFilter={ patternTypeFilter }
+							view={ isGridView ? 'grid' : 'list' }
```

Comment:
Let's keep consistency and provide `isGridView` to the child. And inside the `recordGetAccessEvent` we can do this ternary condition `isGridView ? 'grid' : 'list'`.
```suggestion
							isGridView={ isGridView }
```

Reply:
Good call. That also makes the comment on that `view='grid'` placeholder to satisfy TS unneeded! If we replace `view` with the optional `gridView` for `PatternPreview, we can leave that prop off for the `CategoryGallery` implementation all together. (704c73225e)